Documentation/operations/performance/tuning.rst

```diff
@@ -216,6 +216,27 @@ In order to optimize for maximum performance, Hubble can be disabled:
              --namespace kube-system \\
              --set hubble.enabled=false
 
+You can also choose to stop exposing some event types in which you
+are not interested into. For instance if you are mainly interested into
+dropped traffic, you can disable "trace" events which will likely reduce
+the overall CPU consumption of the agent.
```

Comment:
Please add a warning stating that disabling either event type will impact `cilium monitor`, Hubble observability capabilities, Hubble metrics, and Hubble export.

Reply:
great suggestion, thanks, I've just added it! 👍